app/jobs/create_combined_audio_derivatives_job.rb

```diff
@@ -2,14 +2,18 @@ class CreateCombinedAudioDerivativesJob < ApplicationJob
   def perform(work)
     deriv_creator = CombinedAudioDerivativeCreator.new(work)
     return unless deriv_creator.available_members?
+    sidecar = work.oral_history_content!
+    sidecar.combined_audio_derivatives_job_status = 'started'
+    sidecar.save!
     # Generate the derivatives:
     deriv_info = deriv_creator.generate
     if deriv_info.errors
       Rails.logger.error("Unable to create combined audio derivatives for work #{work.friendlier_id}:")
       Rails.logger.error(deriv_info.errors)
+      sidecar.combined_audio_derivatives_job_status = 'failed'
+      sidecar.save!
       return
```

Comment:
I know this wasn't being done before either -- but what if we actually raise an exception on errors here, so that honeybadger will report it. Then we can set the status to `failed` in an ActiveJob `rescue_from`, like we've done before -- which will also catch other unexpected errors... is that good or bad?

Reply:
Done!